extensions/sql-database-projects/src/models/deploy/deployService.ts

```diff
@@ -0,0 +1,403 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AppSettingType, IDeployProfile, ILocalDbSetting } from './deployProfile';
+import * as UUID from 'vscode-languageclient/lib/utils/uuid';
+import { Project } from '../project';
+import * as constants from '../../common/constants';
+import * as utils from '../../common/utils';
+let fse = require('fs-extra');
+let path = require('path');
+import * as childProcess from 'child_process';
+import * as vscode from 'vscode';
+import * as os from 'os';
+import { ConnectionResult } from 'azdata';
+import * as templates from '../../templates/templates';
+
+interface ValidationResult {
+	errorMessage: string;
+	validated: boolean
+}
+export class DeployService {
+
+	constructor(private _outputChannel: vscode.OutputChannel) {
+	}
+
+	private createConnectionStringTemplate(runtime: string | undefined): string {
+		switch (runtime?.toLocaleLowerCase()) {
+			case 'dotnet':
+				return constants.defaultConnectionStringTemplate;
+				break;
+			// TODO: add connection strings for other languages
+			default:
+				break;
+		}
+		return '';
+	}
+
+	private findAppRuntime(profile: IDeployProfile, appSettingContent: any): string | undefined {
```

Comment:
Can you give this a type besides any? We should avoid using any whenever we can

Reply:
it's what is return from json.parse. the type is any

Reply:
Generally you should be casting that result into a strongly typed interface so that we have type safety within the app at least. (and then optionally verifying that the parsed object actually matches that expected type)